index.js

```diff
@@ -0,0 +1,54 @@
+const bookInput = document.getElementById("bookInput");
+const authorInput = document.getElementById("authorInput");
+const addBtn = document.getElementById("add");
+const error = document.getElementById("error");
+const displayDiv = document.getElementById("display-div");
+let myLibrary = [];
```

Comment:
since you are not reassigning this `myLibrary` array use `const` here to declare it